manim/mobject/svg/brace.py

```diff
@@ -118,14 +119,20 @@ def __init__(
         for mob in mobject, self:
             mob.rotate(angle, about_point=ORIGIN)
 
-    def put_at_tip(self, mob, use_next_to=True, **kwargs):
-        if use_next_to:
-            mob.next_to(self.get_tip(), np.round(self.get_direction()), **kwargs)
-        else:
-            mob.move_to(self.get_tip())
+    def put_at_tip(self, mob, rotate=False, **kwargs):
+
+        if rotate:
             buff = kwargs.get("buff", DEFAULT_MOBJECT_TO_MOBJECT_BUFFER)
-            shift_distance = mob.width / 2.0 + buff
-            mob.shift(self.get_direction() * shift_distance)
+            shift_distance = mob.height / 2.0 + buff
+            mob.move_to(self.get_tip() + self.get_direction() * shift_distance)
```

Comment:
Apart from the change from `next_to` vs `move_to`, what also changes here is how the `shift_distance` is computed. While the old code seemed to have the case of a "vertical" brace in mind (which is oriented horizontally, so to move a mobject away from the tip of the brace you'd need to move it half of its width plus some buffer), this new proposal seems to have "horizontal" braces in mind.

I wonder whether it would be worth to invest some energy into a better, more consistent solution in terms of spacing; but I do think that only very long or very high mobjects are actually affected by this.